### DISCUSSION TITLE
Should we remove the deployment workflows?

### DISCUSSION THREAD

**Discussion by andreivladbrg** [sablier-labs/evm-monorepo#815] [General]
We currently have some workflows in the CI, that are meant to be used to deploy contracts. e.g. https://github.com/sablier-labs/v2-core/blob/main/.github/workflows/deploy-core.yml

<img width="1487" alt="image" src="https://github.com/sablier-labs/v2-core/assets/99738872/02461d3c-425b-45c4-b757-f8b4e058ec7e">

The problems are:
- they have never been used in practive (only to see if they work)
- they are outdated
- maintenance costs
- it is better to deploy on the machine to get the broadcasts
- for those who have warp, we can use drive

IMO they are cool, but not that useful, so I suggest to remove them.

@PaulRBerg what do you say?

**Comment by PaulRBerg** [comment]
Agree, let's remove them.
